src/subgraph/concatenate.c

```diff
@@ -286,13 +217,23 @@ static enum xnn_status setup_concatenate_operator_helper(
   }
 }
 
+static enum xnn_status reshape_concatenaten_operator(
+  struct xnn_operator_data* opdata,
+  struct xnn_value* values,
+  size_t num_values,
+  pthreadpool_t threadpool)
+{
+  return reshape_concatenate_n_operator(opdata, values, num_values, opdata->num_inputs, threadpool);
```

Comment:
Remove the `num_inputs` parameter now that we don't use it It looks like you just don't need these wrappers at all any more.